### DISCUSSION TITLE
set reviewer to last git commit author?

### DISCUSSION THREAD

**Discussion by faenschi** [renovatebot/renovate#16645] [Request Help] [answered]
Hi all

Is it possible to set the reviewer or assignee of a Merge Request to the last git commit author or the last tag creator?

regards angela

**Comment by viceice** [comment, marked as answer]
no

**Comment by HonkingGoose** [comment]
Have you seen these methods to deal with Renovate PR reviewers yet:

1. List reviewers for Renovate PRs only
2. CODEOWNERS file
3. Lottery

I hope you find at least one of these methods helpful. 😉 

## List reviewers for Renovate PRs only

You can tell Renovate to always ask the same people/team for a review with the `reviewers` configuration option. It looks like this:

```json5
{
  "reviewers": ["faenschi", "team-member-1", "other-team-member-2"] // list names
  "reviewers": ["team:foo"] // list a team on GitHub
}
```

## CODEOWNERS file

If you're on GitHub, create a CODEOWNERS file so certain (package)files get reviewed by certain people or teams.

Steps:

1. Discuss with your team who is responsible for what parts of the code
1. Create a `CODEOWNERS` file
1. Set `reviewersFromCodeOwners` to `true` in your Renovate config file
1. when Renovate opens a PR, it checks the `CODEOWNERS` file and ping reviewers as listed in the file

## Lottery

If you want to spread the review work over multiple team members, you can run a lottery:

1. Create an array with `reviewers`
4. Set `reviewersSampleSize` to be the amount of lottery tickets
5. When Renovate opens a PR, it draws the number of lottery tickets, and assigns the winners to review the PR

## Docs

- [`reviewers`](https://docs.renovatebot.com/configuration-options/#reviewers)
- [`reviewersFromCodeOwners`](https://docs.renovatebot.com/configuration-options/#reviewersfromcodeowners)
- [`reviewersSampleSize`](https://docs.renovatebot.com/configuration-options/#reviewerssamplesize)
- [GitHub docs, About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
